content/de/chaos-engineering.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Chaos Engineering
+status: Feedback erwünscht
+category: Konzept
+---
+
+## Was es ist
+
+Chaos Engineering (kurz CE) ist eine experimentbasierte, ganzheitliche Disziplin, um unerwartete Schwachstellen in [verteilten Systeme](/distributed-systems/) in der Produktion zu erkennen, solchen zu widerstehen und das Vertrauen in die Fähigkeiten des Systems zu erhöhen. 
+
+## Welches Problem es löst
+
+Konzepte wie [SRE](/site-reliability-engineering/) und [DevOps](/devops/) konzentrieren sich auf Techniken, die die Produktausfallsicherheit und [Zuverlässigkeit](/reliability/) erhöhen.
+Die Fähigkeit eines Systems, Ausfälle zu tolerieren und gleichzeitig eine angemessene Servicequalität zu gewährleisten, ist in der Regel eine Anforderung an die Softwareentwicklung.
+Es gibt mehrere Facetten, die zu Ausfällen einer Anwendung führen können, z. B. Infrastruktur, Plattform oder andere Komponenten einer ([microservice](/microservices/)-basierten) Anwendung.
+Die häufige Einführung neuer Funktionen in der Produktionsumgebung erhöht die Wahrscheinlichkeit von Ausfallzeiten aufgrund von kritischen Zwischenfällen, die für das Unternehmen erhebliche Folgen haben können.
+
+## Wie es das Problem löst
+
+Chaos-Engineering wird eingesetzt, um die Resilienz von Infrastruktur, Plattform und Anwendungen zu erhöhen.
+Software-Ingenieure, die Chaos-Engineering betreiben, definieren Chaos-Experimente, um proaktiv zufällige Ausfälle zu verursachen. 
+Auf diese Weise kann geprüft werden, ob eine Anwendung, Infrastruktur oder Plattform sich selbst heilen kann und der Ausfall keine spürbaren Auswirkungen auf die Kunden hat.
+Das Ziel dieser Chaos-Experimente ist es, blinde Flecken zu entdecken (z. B. Überwachungs- oder Autoskalierungstechniken) und die Kommunikation zwischen den Teams bei kritischen Ereignissen zu verbessern.
```

Comment:
Dieser und der nächste Satz sprechen nicht darüber, wie Chaos Engineering das Problem löst. Das mit den blinden Flecken haben wir glaube ich quasi auch schon gesagt. Das könnte noch in die entsprechende Stelle mit aufnehmen, aber ich finde Überwachungs- und Autoskalierungstechniken als Beispiele für blinde Flecken irgendwie nicht schlüssig. Bei dem Aspekt mit dem Team-Gefüge bin ich auch nicht ganz sicher. Ist es wirklich Chaos Engineering was da hilft, oder ist es eher so eine allgemeine On-Call, Incident-Management und blameless Post-Mortem Kultur?!